pydis_site/apps/content/resources/guides/python-guides/vps_services.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Free Hosting Services for Discord Bots
+description: "This article covers the disasdvantages of utilising a free hosting service to run a discord bot. Two commonly used services are discussed below. "
```

Comment:
This part, could instead, be moved to a form of introduction that every guide should have.

```suggestion
description: "This article covers the disasdvantages of utilising a free hosting service to run a discord bot."
```